src/lib/vault/network.js

```diff
@@ -0,0 +1,32 @@
+import * as bitcoin from 'bitcoinjs-lib';
+import { Network, NetworkUtil } from 'vault-wallet-toolkit';
+export { Network, NetworkUtil };
+
+// eslint-disable-next-line no-process-env
+const { NET } = process.env;
+
+export const getNetworkEnv = () => {
+  switch (NET) {
+    case 'production':
+      return Network.PRODUCTION;
+    case 'development':
+      return Network.DEVELOPMENT;
+    case 'local':
+      return Network.LOCAL;
+    default:
+      return Network.PRODUCTION;
+  }
+};
+
+export const getBitcoinNetwork = () => {
+  switch (NET) {
+    case 'production':
+      return bitcoin.networks.bitcoin;
+    case 'development':
+      return bitcoin.networks.testnet;
+    case 'local':
+      return bitcoin.networks.regtest;
+    default:
+      return bitcoin.networks.bitcoin;
+  }
+};
```

Comment:
getNetwork from NetworkUtils should return you this afaik

Reply:
👍 This was a leftover before I had `getNetwork` working right. I have fixed this now.